src/shipit_static_analysis/default.nix

```diff
@@ -49,6 +52,26 @@ let
             "MOZ_AUTOMATION" = "1";
           };
         };
+
+        taskRoutes = {
+          "$flatten" = [
+            [
+              # Latest route
+              ("index.project.releng.services.project." + branch + ".shipit_static_analysis.latest")
+
+            ]
+            {
+              "$if" = "firedBy == 'triggerHook'";
```

Comment:
Oh right, so we could apply the same solution to code coverage, by using firedBy == 'schedule' instead. Could you make the changes to code coverage too?